src/EditorFeatures/CSharpTest/GenerateFromMembers/GenerateConstructorFromMembers/GenerateConstructorFromMembersTests.cs

```diff
@@ -1831,5 +1797,187 @@ public async Task TestMissingWhenTypeNotInCompilation()
     </Project>
 </Workspace>");
         }
+
+        [Fact, WorkItem(29198, "https://github.com/dotnet/roslyn/issues/29198")]
+        public async Task TestWithSelectedGetPropertyThatReturnsField1()
+        {
+            await TestInRegularAndScriptAsync(
+@"
+class Program
+{
+    private int _value;
+
+    [|public int Goo
+    {
+        get { return _value; }
+    }|]
+}",
+@"
+class Program
+{
+    private int _value;
+
+    public Program(int value{|Navigation:)|}
```

Comment:
💭  How does this navigation work? I wonder if it is reusable by third-party code fixes, and also wonder if it can handle selections. If so, we could use this feature to resolve the other feature request for `SelectionAnnotation`.

Reply:
You use an annotation and annotate token (so it survives tree transforms).  We take you to the start of that token.